avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastSerializerGeneratorException.java

```diff
@@ -0,0 +1,18 @@
+package com.linkedin.avro.fastserde;
+
+public class FastSerializerGeneratorException extends RuntimeException {
+
+  private static final long serialVersionUID = 1L;
```

Comment:
We don't need this do we? Using Java Serialization is a well-known anti-pattern, so hopefully nothing here relies on that... I'd suggest just removing this.

Reply:
Will remove it.